moped-etl/prefect/tasks/ecs.py

```diff
@@ -344,14 +345,24 @@ def create_task_definition(slug, api_endpoint):
         containerDefinitions=[
             {
                 "name": "graphql-engine",
-                "image": "hasura/graphql-engine:v2.7.0",
+                "image": "hasura/graphql-engine:v2.15.1",
                 # "image": "mendhak/http-https-echo:latest",
                 "cpu": 256,
                 "memory": 512,
                 "portMappings": [
                     {"containerPort": 8080, "hostPort": 8080, "protocol": "tcp"}
                 ],
                 "essential": True,
+                # see: https://github.com/hasura/graphql-engine/issues/1532#issuecomment-1161637925
```

Comment:
tldr: the hasura image does not have `curl` installed, so we have to hack the health check a bit or use a custom image.